### ISSUE TITLE
[Request] Disable mob spawner mobs collision

### ISSUE THREAD

**Issue by TheWranger** [VolmitSoftware/React#361]
Ability to disable collision on mobs that spawn from mob spawners along with a configurable list of mobs that this will effect.
A lot like this plugin but with more configurability: https://www.spigotmc.org/resources/nocollide.24612/

**Comment by cyberpwnn** [comment]
Working on a config for it. How does this look?

```yml
collision:

  # Disables collision for entities about to be culled.
  disable-collide-on-pre-cull: true

  # Disables collision for certain entities based on lag and conditions.
  collision-tweaks-enabled: true
  disable-collide-always:
  - ZOMBIE
  disable-collide-spawner:
  - ELDER_GUARDIAN
  - WITHER_SKELETON
  - STRAY
  - PAINTING
  - FALLING_BLOCK
  - HUSK
  - ZOMBIE_VILLAGER
  - SKELETON_HORSE
  - ZOMBIE_HORSE
  - DONKEY
  - MULE
  - EVOKER
  - VEX
  - VINDICATOR
  - ILLUSIONER
  - MINECART_COMMAND
  - MINECART
  - MINECART_CHEST
  - MINECART_FURNACE
  - MINECART_TNT
  - MINECART_HOPPER
  - MINECART_MOB_SPAWNER
  - CREEPER
  - SKELETON
  - SPIDER
  - GIANT
  - ZOMBIE
  - SLIME
  - GHAST
  - PIG_ZOMBIE
  - ENDERMAN
  - CAVE_SPIDER
  - SILVERFISH
  - BLAZE
  - MAGMA_CUBE
  - ENDER_DRAGON
  - WITHER
  - BAT
  - WITCH
  - ENDERMITE
  - GUARDIAN
  - SHULKER
  - PIG
  - SHEEP
  - COW
  - CHICKEN
  - SQUID
  - WOLF
  - MUSHROOM_COW
  - SNOWMAN
  - OCELOT
  - IRON_GOLEM
  - HORSE
  - RABBIT
  - POLAR_BEAR
  - LLAMA
  - LLAMA_SPIT
  - PARROT
  - VILLAGER
  - FISHING_HOOK
```

**Comment by cyberpwnn** [comment]
Implemented in 6.547
